bl.js

```diff
@@ -107,6 +107,12 @@ BufferList.prototype.get = function get (index) {
 
 
 BufferList.prototype.slice = function slice (start, end) {
+  if (typeof start === 'number' && start < 0) {
```

Comment:
don't need braces for simple single-line `if` statements. `==` is enough for the typeof check too